snap/info.go

```diff
@@ -201,6 +200,12 @@ func (s *Info) NeedsDevMode() bool {
 	return s.Confinement == DevmodeConfinement
 }
 
+// DownlaodInfo contaisn the download URLs of a snap
```

Comment:
typo, Download  also I would say just information (instad of URLS), and that it can be marshalled.